File: Employee.cs

using System.Collections.Generic;
using System;
public class Employee
{
    private List<Expense> _Expenses = new List<Expense>();
    public int ID { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string Position { get; set; }
    public decimal Salary { get; set; }
    public DateTime TerminationDate { get; set; }

    public List<Expense> Expenses
    { get { return _Expenses; } }
}

public class Expense
{
    public string Description { get; set; }
    public decimal Cost { get; set; }
}

File: VB.NET/Employee.vb

Public Class Employee
    Private _Expenses As List(Of Expense) = New List(Of Expense)
    Public Property ID As Integer
    Public Property FirstName As String
    Public Property LastName As String
    Public Property Position As String
    Public Property Salary as Decimal
    Public Property TerminationDate As DateTime

    Public ReadOnly Property Expenses As List(Of Expense)
        Get
            Return _Expenses
        End Get
    End Property
End Class

Public Class Expense
 
    Public Property Description As String
    Public Property Cost As Decimal
End Class
